destination/retry.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/avast/retry-go/v4"
```

Comment:
I am not sure if it's worth incorporating external library for such simple logic.

Reply:
Mainly to save some time. We can easily throw it away later.

Reply:
Removed.